src/backend/main.ts

```diff
@@ -360,6 +361,29 @@ if (!gotTheLock) {
 
     const settings = GlobalConfig.get().getSettings()
 
+    if (settings && settings.analyticsOptIn === true) {
+      const plausible = Plausible()
+      plausible.enableAutoPageviews()
+      const appVersion = app.getVersion()
+      const providersObject = {
+        gog: !!GOGUser.isLoggedIn(),
+        epic: !!LegendaryUser.isLoggedIn(),
+        amazon: !!NileUser.isLoggedIn()
```

Comment:
maybe we can include `sideload` here and count if the number of sideloaded games is > 0? to know how much this feature is used too? maybe even divide by games/urls to have that distinction?